lib/nodes/addon/components/driver-harvester/template.hbs

```diff
@@ -203,6 +203,46 @@
       expand=(action expandFn)
       expandOnInit=true
     }}
+      {{#if loading}}
```

Comment:
@mantis-toboggan-md 

I added a loading icon to avoid the user selecting the namespace/topology key when the node/namespace is not loaded.